src/components/composer/composer/composer.ts

```diff
@@ -197,7 +200,9 @@ export class Composer extends Component<ComposerProps, SpreadsheetChildEnv> {
         assistantStyle.right = `0px`;
       }
     }
-    return cssPropertiesToCss(assistantStyle);
+
+    const assistantCss: string = cssPropertiesToCss(assistantStyle);
+    return assistantCss + this.props.composerAssistantStyle;
```

Comment:
I didn't read the composer code in much details so I might be wrong, but can't the max-height in topBar composer be acheived with the existing props (`props.delimitation`, `props.rect`) ? The gridComposer seems to only use them and be fine

Reply:
Currently, the top bar doesn't send these props, but I believe it should send `demarcation'. I need to include this send in the props and set the max height value here, as that is the exact value I was trying to get before.